docs/src/content/docs/ko/reference/configuration.mdx

```diff
@@ -571,3 +572,13 @@ starlight({
 ```
 
 나만의 플러그인을 만드는 방법에 대한 자세한 내용은 [플러그인 참조](/ko/reference/plugins/)를 확인하세요.
+
+### `credits`
+
+사이트 바닥글에 “Built with Starlight” 링크 표시를 활성화합니다.
```

Comment:
Just adding a note that the text could be updated later when the [associated UI string](https://github.com/withastro/starlight/blob/main/packages/starlight/translations/ko.json#L28) is translated.